### DISCUSSION TITLE
How to update on docker ?

### DISCUSSION THREAD

**Discussion by toutjavascript** [open-webui/open-webui#1232] [Q&A]
Hello

Thanks for this smart tool
How can we update the release on docker ?

**Comment by gibru** [comment]
[Does this answer your question?](https://github.com/open-webui/open-webui?tab=readme-ov-file#keeping-your-docker-installation-up-to-date)

**Reply by justinh-rahb** [reply]
In addition to the above we have an [expanded updating document](https://docs.openwebui.com/getting-started/updating) on the docs site.

**Reply by toutjavascript** [reply]
Thanks a lot.
It is easy now

**Comment by pr0fsmith** [comment]
I've followed the update instructions several times here https://docs.openwebui.com/getting-started/updating but my instance won't update. I'm still stuck at 0.4.8.  Any ideas?
